webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -139,6 +149,46 @@ class ManageManifestModal extends Component {
                   <React.Fragment>
                     <Grid>
                       <h3>{__('Subscription Manifest')}</h3>
+                      {manifestExpiringSoon &&
+                        <Alert
+                          ouiaId="manifest-expiring-soon-alert"
+                          variant="warning"
+                          title={__('Manifest expiring soon')}
+                        >
+                          <FormattedMessage
+                            defaultMessage={__('Your manifest will expire in {daysMessage}. To extend the expiration date, refresh your manifest or import a new manifest.')}
```

Comment:
Refreshing will work for connected users. For disconnected users, you'll have to re-export from the Customer Portal and re-import on the disconnected instance, but I couldn't think of a better way to word this.

Reply:
Should we maybe change the wording to:

`To extend the expiration date, refresh your manifest or import a new manifest if you are running a disconnected setup.`